test/baseline/databases/metadata/metadata_0_03.txt

```diff
@@ -199,7 +199,7 @@ GetScalars(0).minDataExtents = 0
 GetScalars(0).maxDataExtents = 0
 GetScalars(0).matRestricted = ()
 GetScalars(0).treatAsASCII = 0
-GetScalars(0).enumerationType = GetScalars(0).None  # None, ByValue, ByRange, ByBitMask, ByNChooseR
+GetScalars(0).enumerationType = GetScalars(0).NONE  # NONE, ByValue, ByRange, ByBitMask, ByNChooseR
```

Comment:
Am curious about all these txt rebaselines...another way to go is to adjust the text filtering we use in our testing to handle this and allow `.None` and `None,` to match `.NONE` and `NONE,`. That might have other positive benefits for anyone using our test suite for text testing too.

Reply:
Too much magic for me, could fail in unintended ways, but lets talk about it at a meeting if you want to explore this more.

Reply:
Sure. No worries.